core/src/components/select/readme.md

```diff
@@ -938,6 +938,16 @@ Type: `Promise<any>`
 
 
 
+## Shadow Parts
```

Comment:
Do we want these to be documented yet?

Reply:
they documented already in the source code, for ion-button renamed `@part` to @TODOpart` to skip it, i can do the same for select i guess

Reply:
I just updated it, sorry didn't see this comment first